javascript-data-model/order-history.js

```diff
@@ -0,0 +1,79 @@
+var orderHistory = [
+  {
+    orderPlaced: 'Aug 4, 2020',
+    total: 34.00,
+    orderNumber: '114-3941689-8772232',
+    delivered: 'Aug 8, 2020',
+    shipTo: 'JS Masher',
+    product: [
+      {
+        title: 'Javascript for impatient programmers',
+        author: 'Rauschmayer, Dr. Axel',
+        returnWindow: 'Sept 7, 2020',
+        price: 31.55
+      }
+    ]
+  },
+
+  {
+    orderPlaced: 'Jul 19, 2020',
+    total: 44.53,
+    orderNumber: '113-998426-1280257',
+    delivered: 'Jul 20, 2020',
+    shipTo: 'JS Masher',
+    product: [
+      {
+        title: 'The Timeless Way of Building',
+        author: 'Alexander, Christopher',
+        returnWindow: 'Aug 19, 2020',
+        price: 41.33
+      }
+    ]
+  },
+
+  {
+    orderPlaced: 'Jul 4, 2020',
+    total: 17.22,
+    orderNumber: '114-2875557-9059409',
+    delivered: 'Jul 7, 2020',
+    shipTo: 'JS Masher',
+    product: [
+      {
+        title: 'Gamecub Controller Adapter. Super Smash Bros Switch Gamecube Adapted for WII U, PC, Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+        returnWindow: 'Aug 5, 2020',
+        price: 15.86
+      }
+    ]
+  },
+
+  {
+    orderPlaced: 'July 3, 2020',
+    total: 138.93,
+    orderNumber: '113-2883177-2648248',
+    delivered: 'Jul 5, 2020',
+    shipTo: 'JS Masher',
+    product: [
+      {
+        title: 'GameCube Controller - Super Smash BroadcastChannel. Edition (Ninentedo Switch)',
+        returnWindow: 'Aug 4, 2020',
+        price: 94.95
+      }
+    ]
+  },
+
+  {
+    orderPlaced: 'Jul 5, 2020',
+    total: 138.93,
+    orderNumber: '113-2883177-2648248',
+    delivered: 'Jul 5, 2020',
+    shipTo: 'JS Masher',
+    product: [
+      {
+        title: 'The Art of Sql',
+        author: 'Faroult, Stephanie',
+        returnWindow: 'Aug 4, 2020',
+        price: 33.99
+      }
+    ]
+  }
```

Comment:
This looks like two separate orders, but it is only one order. You should have one order object with two objects stored in the `product` array, one for each product ordered